fitbenchmarking/fitbenchmarking
# Fix options_template and add tests for usage examples

#### Description of Work

Fixes #773 


#### Testing Instructions

1. Check that all usage examples can be run without errors
2. Check that tests for usage examples pass
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
